README.md

```diff
@@ -91,7 +91,13 @@ $ brew install lima
 - On ARM hosts, QEMU has to be patched for enabling `--accel=hvf` support. See https://gist.github.com/nrjdalal/e70249bb5d2e9d844cc203fd11f74c55
 
 - Download the binary archive of Lima from https://github.com/lima-vm/lima/releases ,
-and extract it under `/usr/local` (or somewhere else).
+and extract it under `/usr/local` (or somewhere else). For instance, to extract it under `$HOME`:
+
+```bash
+cd ~
+VERSION=$(curl --silent "https://api.github.com/repos/lima-vm/lima/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/' | cut -c 2-)
+wget -c https://github.com/lima-vm/lima/releases/download/v$VERSION/lima-$VERSION-`uname`-`uname -m`.tar.gz -O - | tar -xz
```

Comment:
Seems lacing `-C` option of `tar`

Reply:
It seems -x (extract) and `-c(create) -C <dir>` are mutual exclusion.

```bash
~/dev/temp/test
❯ tar --help
tar(bsdtar): manipulate archive files
First option must be a mode specifier:
  -c Create  -r Add/Replace  -t List  -u Update  -x Extract
Common Options:
  -b #  Use # 512-byte records per I/O block
  -f <filename>  Location of archive
  -v    Verbose
  -w    Interactive
Create: tar -c [options] [<file> | <dir> | @<archive> | -C <dir> ]
  <file>, <dir>  add these items to archive
  -z, -j, -J, --lzma  Compress archive with gzip/bzip2/xz/lzma
  --format {ustar|pax|cpio|shar}  Select archive format
  --exclude <pattern>  Skip files that match pattern
  -C <dir>  Change to <dir> before processing remaining files
  @<archive>  Add entries from <archive> to output
List: tar -t [options] [<patterns>]
  <patterns>  If specified, list only entries that match
Extract: tar -x [options] [<patterns>]
  <patterns>  If specified, extract only entries that match
  -k    Keep (don't overwrite) existing files
  -m    Don't restore modification times
  -O    Write entries to stdout, don't restore to disk
  -p    Restore permissions (including ACLs, owner, file flags)
bsdtar 3.3.2 - libarchive 3.3.2 zlib/1.2.11 liblzma/5.0.5 bz2lib/1.0.6

~/dev/temp/test
❯ wget -c https://github.com/lima-vm/lima/releases/download/v$VERSION/lima-$VERSION-`uname`-`uname -m`.tar.gz -O - | tar -xzc -C .
tar: Can't specify both -c and -x
```